File: day2_select.cpp
#include<iostream>
using namespace std;

void selection_sort(int *arr);
void display(int *arr);

int main()
{
    
    int arr[10];
    for(int i=0;i<10;i++)
    {
        arr[i]=rand()%11;
    }
    cout<<"The array before sorting is"<<endl;
    display(arr);
    selection_sort(arr);
    

}
void display(int *arr)
{
    
    for(int i=0;i<10;i++)
    {
        cout<<arr[i]<<"\t";
    }
cout<<endl;
}
void selection_sort(int *arr)
{
    int min;
    for(int i=0;i<10;i++)
    {
        min=arr[i];
        for(int j=i+1;j<10;j++)
        {
            if(arr[j]<arr[i])
            {
                swap(arr[j],arr[i]);
            }
        }
    }
    cout<<"The array after sorting is"<<endl;
    display(arr);
    
}

File: day2_select.py
import random
arr=[]

def sort_elm(arr):
    for i in range(n):
        for j in range (i+1,n):
            if(arr[j]<arr[i]):
                arr[i],arr[j]=arr[j],arr[i]
    
def display_elm(arr):
    for i in range(n):
        print(arr[i],end=" ")
    print(end="\n")



n=int(input("Enter the length of an array"))

for i in range(n):
    num=random.randrange(0,11,2)
    arr.append(num)

print("The orignal array is ")
display_elm(arr)
sort_elm(arr)
print("The array after sorting is")
display_elm(arr)


